libs/ledger-live-common/src/types/index.ts

```diff
@@ -1,13 +1,5 @@
 export * from "../derivation";
-export * from "./currencies";
-export * from "./account";
-export * from "./operation";
-export * from "./portfolio";
 export * from "../account/groupOperations";
 export * from "../account/accountId";
-export * from "./transaction";
 export * from "../generated/types";
-export * from "./bridge";
-export * from "./pagination";
-export * from "./nft";
 export * from "../featureFlags/types";
```

Comment:
Done, I could not delete the remaining types in derivation because there are also types in hw/getAddress and i didn't know what to do with them. Also we need to figure out how to treat the DerivationMode type